yarn.lock

```diff
@@ -22753,7 +22753,12 @@ typescript-eslint@^7.5.0:
     "@typescript-eslint/parser" "7.18.0"
     "@typescript-eslint/utils" "7.18.0"
 
-typescript@5.0.4, typescript@~5.0.4:
+typescript@5.2.2:
+  version "5.2.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-5.2.2.tgz#5ebb5e5a5b75f085f22bc3f8460fba308310fa78"
+  integrity sha512-mI4WrpHsbCIcwT9cF4FZvr80QUeKvsUsUvKDoR+X/7XHQH98xYD8YHZg7ANtz2GtZt/CBq2QJ0thkGJMHfqc1w==
+
+typescript@~5.0.4:
```

Comment:
this is not good, what's the origin of this dupe ? can we resolve it ? 

I don't see API-extractor being bumped in this PR , please make sure to do that in order to align TS versions

Reply:
API extractor next version will have TS 5.3.3

Reply:
another unfortunate thing :D yeah resolution it is ty !